templates/BenchmarkDotNet.BenchmarkPorjectTemplate.CSharp/content/.template.config/template.json

```diff
@@ -0,0 +1,86 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "identity": "BenchmarkDotNet.BenchmarkProjectTemplate.CSharp",
+    "name": "Benchmark Project",
+    "shortName": "benchmark",
+    "groupIdentity": "BenchmarkDotNet",
+    "classifications": [
+      "Benchmark",
+      "Library"
+    ],
+    "precedence": "3000",
+    "tags": {
+      "language": "C#",
+      "type": "project"
+    },
+    "description": "A project template for creating benchmarks.",
+    "author": "BenchmarkDotnet",
+    "generatorVersions": "[1.0.0.0-*)",
+    
+    "sourceName": "_BenchmarkProjectName_",
+    "preferNameDirectory": true,
+    "defaultName": "MyBenchmark",
+    "symbols": {
+      "benchmarkName": {
+        "type": "parameter",
+        "datatype": "string",
+        "description": "The name of the benchmark class.",
+        "defaultValue": "MyBenchmark",
+        "FileRename" : "_BenchmarkName_",
+        "replaces": "$(BenchmarkName)"
+      },
+      "framework": {
+        "type": "parameter",
+        "datatype": "string",
+        "description": "The target framework for the project.",
+        "defaultValue": "netstandard2.0",
```

Comment:
What if they want to target multiple frameworks like ``netstandard2.0;netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;netcoreapp3.0;net460;net461;net462;net47;net471;net472``?
I think this template should be aware of said ``;`` separated frameworks and also replace the ``<TargetFramework>$(Framework)</TargetFramework>`` of the xml with ``<TargetFrameworks>$(Framework)</TargetFrameworks>``

Reply:
Hmm, basically I like the idea. On the other hand, I'm not sure if the user would understand how it behaves when the toolchains are explict set by attributes.

@adamsitnik What do you think is the best way? Has the attribute priority over the `csproj` configuration when BenchmarkDotNet is generating the benchmark project?